### DISCUSSION TITLE
NFC Usage for both versions | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#969] [Q&A]
# NFC Usage for both versions | Seeed Studio Wiki

NFC Usage for XIAO nRF52840 (Sense)

https://wiki.seeedstudio.com/XIAO-BLE-Sense-NFC-Usage/

**Comment by welllrafael** [comment]
Any updates about Seeed Studio XIAO nRF52840 board ?

**Reply by EdgarMoon** [reply]
Hello, 
  I  also tried find a way to use NFC back in time. Finally, I can use NFC. I just checked.

  I used 2.9.2 version of Seeed nRF52 mbed-enabled Board. NFC example included in that version. 

  Also, you need to add some capacitance on NFC Pins to work.

  Please refer to link below, There is picture that showing how to add Caps on the board.

  https://forum.seeedstudio.com/t/xiao-ble-nfc-doesnt-work/264543/13
 
  It works on my Seeed board. So, I hope it works as well to you

**Reply by andriandreo** [reply]
# USER UPDATE

I managed to replicate the example code above using the "Seeed nRF52 mbed-enabled Boards Library" as stated and both antennas below:

- [MOLEX 1462360051](https://es.farnell.com/en-ES/molex/146236-0051/nfc-antenna-13-56mhz-adhesive/dp/3292214)
- [TAOGLAS FXR.07.A.DG](https://es.farnell.com/en-ES/taoglas/fxr-07-a-dg/rf-antenna-nfc-13-56mhz-adhesive/dp/3678658)

Both worked right just after soldering without further circuit tuning. Later, I added two 43 pF tuning caps from each NFC pad to GND to further enhance the interrogation distance for the [TAOGLAS FXR.07.A.DG](https://es.farnell.com/en-ES/taoglas/fxr-07-a-dg/rf-antenna-nfc-13-56mhz-adhesive/dp/3678658). 

The ~40 pF cap value was calculated from subsection "6.14.10 NFCT antenna recommendations" in [nRF52840 MCU product specification datasheet](https://docs.nordicsemi.com/bundle/nRF52840_PS_v1.8/resource/nRF52840_PS_v1.8.pdf).

---
Pics along with this info can be also found in the Seeed Studio forum topic below:

https://forum.seeedstudio.com/t/xiao-nrf52840-nfc-antenna-insights/277696

**Reply by MatthewJeffson** [reply]
Wow! Thank you guys for providing the updates!
Best Regards,
Matthew

**Comment by andriandreo** [comment]
BTW, they mention that `"The Seeed nRF52 Boards version 1.1.3 [...] have been tested and approved"`, but sketch fails to build because `NFCT.h` is not found anywhere for `non-mbed` BSP.

**Reply by chbeeler** [reply]
I agree with andriandreo. Why does NFC not work with the non-mbed version? Can someone help to fix that?

**Reply by Lukilyy** [reply]
Hello! Thank you for your feedback regarding the NFC function of XIAO nRF52840 (Sense). We will conduct functional tests in the future and will inform you of the results as soon as they are available.

**Comment by orberec** [comment]
Does the board reads NFC cards?

**Reply by nfs0619** [reply]
Hello, the NFC module can only passively respond to requests from an external reader (e.g., sending a text string to a smartphone) but cannot actively read other NFC tags.
